src/core/Form/Radiobutton/__snapshots__/Radiobutton.test.tsx.snap

```diff
@@ -0,0 +1,2033 @@
+  position: absolute;
+  margin: 8px 0 0 3px;
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input + label {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+  min-height: 27px;
+  line-height: 1.5em;
+  padding-left: 26px;
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input + label::before {
+  content: '';
+  position: absolute;
+  display: inline-block;
+  left: 0;
+  top: 5px;
+  width: 16px;
+  height: 16px;
+  border-radius: 50%;
+  border: 1px solid hsl(202,7%,67%);
+  background: transparent;
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input + label::after {
+  content: '';
+  position: absolute;
+  display: inline-block;
+  left: 5px;
+  top: 10px;
+  border-radius: 50%;
+  width: 8px;
+  height: 8px;
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input:checked + label:before {
+  border: 1px solid hsl(212,63%,45%);
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input:checked + label:after {
+  background: hsl(212,63%,45%);
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input:focus + label::before {
+  outline: 0;
+  border-radius: 2px;
+  box-shadow: 0 0 0 2px hsl(196,77%,55%);
+  border-radius: 50%;
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input:focus + label::before:not(:focus-visible) {
+  box-shadow: none;
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input:disabled + label {
+  cursor: not-allowed;
+  color: hsl(202,7%,67%);
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input:disabled + label:before {
+  border: 1px solid hsl(202,7%,80%);
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input:disabled + label:after {
+  background: hsl(202,7%,97%);
+  width: 16px;
+  height: 16px;
+  top: 6px;
+  left: 1px;
+}
+
+.c1.fi-radiobutton .fi-radiobutton_input:disabled:checked + label:after {
+  background: hsl(202,7%,67%);
+  width: 8px;
+  height: 8px;
+  top: 10px;
+  left: 5px;
+}
+
+.c1.fi-radiobutton--disabled .fi-radiobutton_hintText {
+  color: hsl(202,7%,67%);
+}
+
+.c1.fi-radiobutton--large .fi-radiobutton_hintText {
+  padding-left: 40px;
+}
+
+.c1.fi-radiobutton--large .fi-radiobutton_input {
+  margin: 10px 0 0 9px;
+}
+
+.c1.fi-radiobutton--large .fi-radiobutton_input + label {
+  padding-left: 40px;
+  padding-top: 2px;
+  min-height: 34px;
+}
+
+.c1.fi-radiobutton--large .fi-radiobutton_input + label::before {
+  left: 0px;
+  top: 2px;
+  width: 28px;
+  height: 28px;
+}
+
+.c1.fi-radiobutton--large .fi-radiobutton_input + label::after {
+  left: 7px;
+  top: 9px;
+  width: 16px;
+  height: 16px;
+}
+
+.c1.fi-radiobutton--large .fi-radiobutton_input:disabled + label:after {
+  width: 28px;
+  height: 28px;
+  top: 3px;
+  left: 1px;
+}
+
+.c1.fi-radiobutton--large .fi-radiobutton_input:disabled:checked + label:after {
+  background: hsl(202,7%,67%);
+  width: 16px;
+  height: 16px;
+  top: 9px;
+  left: 7px;
+}
+
+<div>
+  <div
+    class="c0 fi-radiobutton_container fi-radiobutton c1 fi-radiobutton--disabled"
+  >
+    <input
+      class="c2 fi-radiobutton_input"
+      disabled=""
+      id="test-id"
+      type="radio"
+      value=""
```

Comment:
Same goes here, having empty string as value might have side-effects.

Reply:
Yes, good finds. These should be changed like it was done in other PR with the **Checkbox**'s value:  https://github.com/vrk-kpa/suomifi-ui-components/pull/375/commits/66d14aa9711ba03a8f4cb80ee653676ea5895d16

Reply:
This change is now committed to repository.